src/pages/components/file-uploader/accessibility.mdx

```diff
@@ -19,82 +19,51 @@ import {
 
 <PageDescription>
 
-The file uploader React Carbon component has been tested against the latest
-[W3C Web Content Accessibility Guidelines (WCAG) 2.1 Level A and AA success criteria](https://www.w3.org/TR/WCAG21/)
-and violations have been identified as high priority issues. This document will
-be updated when these accessibility issues are resolved.
+No accessibility annotations are needed for file uploaders, but keep these considerations in mind if you’re modifying Carbon or creating a custom component.
```

Comment:
> No accessibility annotations are needed for breadcrumbs, but keep these considerations in mind if you are modifying Carbon or creating a custom component.

- I noticed you have a contraction for "you’re" on the File uploader a11y tab in this PR and on the Slider tab but not on the Breadcrumb tab and Code snippet tab. I would just keep it consistent across the board for this same sentence in all places it shows up.

Reply:
Nice catch. I think Anna asked for the contractions to be removed in some of the reviews. I did point out to her that Carbon uses the "you're" contraction all over the place (57 occurrences in 35 files), so I think we can just make a decision and stick with it.
Do you have a preference?

Reply:
made change across accessibility content I've updated.